common.sh

```diff
@@ -216,3 +213,14 @@ fi
 
 # Defaults the variable to enable testing a custom machine-api-operator image
 export TEST_CUSTOM_MAO=${TEST_CUSTOM_MAO:-false}
+
+# mirror images for installation in restricted network
+export MIRROR_IMAGES=${MIRROR_IMAGES:-}
+
+# variables for local registry configuration
+export LOCAL_REGISTRY_ADDRESS=${LOCAL_REGISTRY_ADDRESS:-"192.168.111.1"}
+export LOCAL_REGISTRY_PORT=${LOCAL_REGISTRY_PORT:-"5000"}
+export REGISTRY_USER=${REGISTRY_USER:-ocp-user}
+export REGISTRY_PASS=${REGISTRY_PASS:-ocp-pass}
+export REGISTRY_DIR=${REGISTRY_DIR:-/opt/registry}
```

Comment:
/opt commonly fills up the root filesystem with default partitioning schemes, and many folks are using WORKING_DIR to point to some other location with more space, so I wonder if we should default this to e.g `${WORKING_DIR}/registry` ?

This would likely avoid breaking folks who have already set WORKING_DIR due to lack of space on / ?

Reply:
That makes sense. Updated, thanks!